test/integration/marked-spec.js

```diff
@@ -1,17 +1,451 @@
+  var section = 'Lists';
+
+  // var shouldPassButFails = [];
+  var shouldPassButFails = [282, 270, 280, 278, 271, 272, 273, 275, 274, 264, 277, 265, 276, 279, 267, 269];
+
+  var willNotBeAttemptedByCoreTeam = [];
+
+  var ignore = shouldPassButFails.concat(willNotBeAttemptedByCoreTeam);
+
+  cmSpec.forEach(function(spec) {
+    messenger.test(spec, section, ignore);
+  });
+});
+
+describe('CommonMark 0.28 Inlines', function() {
+  var section = 'Inlines';
+
+  // var shouldPassButFails = [];
+  var shouldPassButFails = [];
+
+  var willNotBeAttemptedByCoreTeam = [];
+
+  var ignore = shouldPassButFails.concat(willNotBeAttemptedByCoreTeam);
+
+  cmSpec.forEach(function(spec) {
+    messenger.test(spec, section, ignore);
+  });
 });
 
-// http://spec.commonmark.org/0.28/#example-230
-it('should start an ordered list at 0 when requested', function () {
-  expect(marked('0. ok'))
-    .toBe('<ol start="0">\n<li>ok</li>\n</ol>\n')
+describe('CommonMark 0.28 Backslash escapes', function() {
+  var section = 'Backslash escapes';
+
+  // var shouldPassButFails = [];
+  var shouldPassButFails = [290, 291, 289, 293, 297, 301, 299, 298, 300];
+
+  var willNotBeAttemptedByCoreTeam = [];
+
+  var ignore = shouldPassButFails.concat(willNotBeAttemptedByCoreTeam);
+
+  cmSpec.forEach(function(spec) {
+    messenger.test(spec, section, ignore);
+  });
 });
 
-// http://spec.commonmark.org/0.28/#example-234
-it('indents code within an explicitly-started ordered list', function () {
-  expect(marked('  10.  foo\n\n           bar'))
-    .toBe('<ol start="10">\n<li><p>foo</p>\n<pre><code>bar\n</code></pre></li>\n</ol>\n');
+describe('CommonMark 0.28 Entity and numeric character references', function() {
+  var section = 'Entity and numeric character references';
+
+  // var shouldPassButFails = [];
+  var shouldPassButFails = [311, 309, 310, 308];
+
+  var willNotBeAttemptedByCoreTeam = [];
+
+  var ignore = shouldPassButFails.concat(willNotBeAttemptedByCoreTeam);
+
+  cmSpec.forEach(function(spec) {
+    messenger.test(spec, section, ignore);
+  });
+});
+
+describe('CommonMark 0.28 Code spans', function() {
+  var section = 'Code spans';
+
+  // var shouldPassButFails = [];
+  var shouldPassButFails = [330, 316, 325, 327, 328, 320, 323, 322];
+
+  var willNotBeAttemptedByCoreTeam = [];
+
+  var ignore = shouldPassButFails.concat(willNotBeAttemptedByCoreTeam);
+
+  cmSpec.forEach(function(spec) {
+    messenger.test(spec, section, ignore);
+  });
+});
+
+describe('CommonMark 0.28 Emphasis and strong emphasis', function() {
+  var section = 'Emphasis and strong emphasis';
+
+  // var shouldPassButFails = [];
+  var shouldPassButFails = [447, 450, 445, 409, 448, 444, 395, 402, 443, 396, 398, 403, 372, 389, 456, 404, 371, 420, 455, 422, 387, 414, 452, 458, 421, 457, 424, 451, 426, 423, 416, 454, 349, 347, 453, 346, 353, 432, 378, 428, 369, 332, 381, 431, 363, 359, 380, 360, 355, 356, 342, 388, 364, 368, 433, 435, 434, 392, 348, 382, 436, 333, 449, 427, 377, 334];
```

Comment:
add numbers 352, 354, 393, 394, and 419 to skip the failing tests

Reply:
If CI does something different from my local environment, I'm not gonna be able to trust my local, you know? Is there a reason this is happening?